action.yml

```diff
@@ -56,6 +56,9 @@ inputs:
   revision-provisioned:
     required: false
     description: 'Minimum count of provisioned instances'
+  connectivity:
```

Comment:
1. The input name does not follow the pattern when all inputs related to revision start with the `revision-` prefix.
2. It is not clear, either from the input's name or from the description, what kind of value is expected here. I think `revision-network-id` would be a much more clear name.

Reply:
fixed